bigtable-client-core-parent/bigtable-client-core/src/test/java/com/google/cloud/bigtable/grpc/async/TestMutateRowsRequestManager.java

```diff
@@ -91,7 +89,6 @@ private static ProcessingStatus send(
 
   @Before
   public void setup() {
-    MockitoAnnotations.initMocks(this);
```

Comment:
does this need `@Rule`? (similar to implementation in next file)

Reply:
nope, this class doesn't use `@Mock` annotations

Reply:
got it, thanks!